core/src/test/java/org/apache/accumulo/core/client/lexicoder/SequenceLexicoderTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.client.lexicoder;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeSet;
+
+import org.apache.accumulo.core.clientImpl.lexicoder.AbstractLexicoderTest;
+import org.apache.accumulo.core.util.TextUtil;
+import org.apache.hadoop.io.Text;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SequenceLexicoderTest extends AbstractLexicoderTest {
+
+  private final List<Long> data0 = new ArrayList<>();
+  private final List<Long> data1 = new ArrayList<>();
+  private final List<Long> data2 = new ArrayList<>();
+  private final List<Long> data3 = new ArrayList<>();
+  private final List<Long> data4 = new ArrayList<>();
+  private final List<Long> data5 = new ArrayList<>();
+
+  @Before
+  public void setUp() {
```

Comment:
Would be nice if the data had empty elements in order to test the sort order of list with empty elements vs empty list.

Reply:
I just copied `ListLexicoder` test and added an empty list to the data sets. I can add a case for empty elements, but it will require changing the element type from `Long` to something with variable since like `String`.

Reply:
Thanks for doing this in the new PR